conf/mini_provisioner/v2/ha.cleanup.tmpl.3-node

```diff
@@ -10,13 +10,33 @@ cortx:
       secret: TMPL_SECRET
       user: TMPL_USER
 server_node:
-  TMPL_MACHINE_ID:
-    cluster_id: TMPL_CLUSTER_ID
+  TMPL_MACHINE_ID_1:
+    cluster_id: TMPL_CLUSTER_ID_1
     network:
       data:
-        private_fqdn: TMPL_PRIVATE_FQDN
+        private_fqdn: TMPL_PRIVATE_FQDN_1
       management:
-        interfaces: TMPL_INTERFACES
-        netmask: TMPL_NETMASK
+        interfaces: TMPL_INTERFACES_1
+        netmask: TMPL_NETMASK_1
```

Comment:
I think the interfaces and netmask value will be common across all nodes. If yes, we could put just TMPL_INTERFACES and TMPL_NETMASK in all template files.

Reply:
Done